.github/workflows/iroha2-base.yml

```diff
@@ -18,14 +18,8 @@ jobs:
     if: github.ref == 'refs/heads/iroha2-${{ matrix.tag }}'
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v3
```

Comment:
Checkout is implied by [`docker/build-push-action`](https://github.com/docker/build-push-action)